config.local.yml

```diff
@@ -48,7 +48,10 @@ rest_runner:
   - device: cpu
     transcription_enabled: True
     # model: tiny, small, medium, large, large-v3, large-v3-turbo (https://huggingface.co/Systran)
-    model: large-v3-turbo
+    models: 
```

Comment:
why in .local. different than in non local?

Reply:
Honestly I didnt see any use in supporting the large or medium model in the local scenario. Locally I think using the best or fastest models are the only reasonable use cases.